Dockerfile

```diff
@@ -1,11 +1,7 @@
-FROM debian:jessie
+FROM resin/amd64-systemd:latest
```

Comment:
resin-base is using Node v4.2.6. I will need to update amd64-node base image first then we can change to that base image later